src/main/java/org/mule/module/json/internal/JsonSchemaValidator.java

```diff
@@ -285,6 +288,11 @@ private JsonSchemaValidator(JsonSchema schema, ObjectMapper objectMapper) {
    * @param input the json to be validated
    */
   public void validate(InputStream input) {
+
+    if (Boolean.parseBoolean(System.getProperty(VALIDATOR_FAIL_ON_TRAILING_TOKENS))) {
```

Comment:
Shouldn’t you use System.getProperty(VALIDATOR_FAIL_ON_TRAILING_TOKENS, false) to set a default value?

Reply:
the getProperty only take String as parameters. And puts a default value when the property value is null. But the Boolean.parseBoolean(null) is false. So I think it's redundant. It should be:
`(Boolean.parseBoolean(System.getProperty(VALIDATOR_FAIL_ON_TRAILING_TOKENS, "false")))`